compiler/noirc_frontend/src/elaborator/mod.rs

```diff
@@ -872,15 +875,6 @@ impl<'context> Elaborator<'context> {
             | Type::Forall(..)
             | Type::Error => (),
         }
-
-        if let Type::Alias(alias_type, generics) = typ {
-            self.mark_parameter_type_as_used(&alias_type.borrow().get_type(generics));
-            return;
-        }
-
-        if let Type::Struct(struct_type, _generics) = typ {
```

Comment:
Deleted both of these `if let` statements as they are redundant to the match statement above.